release/ci-steps/doc-new-release-notes.mjs

```diff
@@ -0,0 +1,142 @@
+  });
+  apimSidebarFolders.get(releaseNotesIndex).set('folderitems', releaseNotesItems);
+
+  fs.writeFileSync(apimSidebarFile, apimSidebarDoc.toString());
+  await $`git add ./${apimSidebarFile}`;
+  fs.appendFileSync(
+    `${docApimReleaseNotesFile}`,
+    `:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/changelog-${versions.trimmed}.html
+:page-folder: apim
+:page-toc: false
+:page-layout: apim3x
+
+= APIM ${versions.trimmed} changelog
+
+For upgrade instructions, please refer to https://docs.gravitee.io/apim/3.x/apim_installguide_migration.html[APIM Migration Guide]
+
+*Important:* If you plan to skip versions when you upgrade, ensure that you read the version-specific upgrade notes for each intermediate version. You may be required to perform manual actions as part of the upgrade.
+
+// NOTE: Global ${versions.trimmed} release info here
+
+// <DO NOT REMOVE THIS COMMENT - ANCHOR FOR FUTURE RELEASES>
+`,
+  );
+}
+
+let releaseNotesPatchTemplate = `
+== APIM - ${releasingVersion} (${new Date().toISOString().slice(0, 10)})
+
+=== Gateway
+
+// TODO: List all Bug fixes & Improvements
+
+=== API
+
+// TODO: List all Bug fixes & Improvements
+
+=== Console
+
+// TODO: List all Bug fixes & Improvements
+
+=== Portal
+
+// TODO: List all Bug fixes & Improvements
+`;
+echo(releaseNotesPatchTemplate);
+
+// write after anchor
+const releaseNotesFileContent = fs.readFileSync(docApimReleaseNotesFile, 'utf8');
+const releaseNotesFileContentWithPatch = releaseNotesFileContent.replace(
+  '// <DO NOT REMOVE THIS COMMENT - ANCHOR FOR FUTURE RELEASES>',
+  `$&
+ ${releaseNotesPatchTemplate}`,
+);
+fs.writeFileSync(`${docApimReleaseNotesFile}`, releaseNotesFileContentWithPatch);
+
+echo(chalk.blue(`# Commit and push changelog to ${gitBranch}`));
+try {
+  await $`gh pr close --delete-branch ${gitBranch}`;
+} catch (e) {
+  // Best effort to have no open PR before creating a new one
+}
+await $`git checkout -b ${gitBranch}`;
+await $`git add ./${docApimReleaseNotesFile}`;
+await $`git commit -m "chore: add changelog for ${releasingVersion}"`;
+await $`git push --set-upstream origin ${gitBranch}`;
+
+const prBody = `
+# New APIM version ${releasingVersion} has been released
+📝 You can modify the changelog template online [here](https://github.com/gravitee-io/gravitee-docs/edit/${gitBranch}/${docApimReleaseNotesFile})
+
+Here is some information to help with the writing: 
+
+## Commit messages 
+<details>
+  <summary>See all feats commit</summary>
+
+${allFeatCommits.stdout}
+
+</details>
+
+<details>
+  <summary>See all fixs commit</summary>
+
+${allFixCommits.stdout}
+</details>
+
+## Jira issues
+
+[See all Jira issues for ${versions.branch} version](https://gravitee.atlassian.net/jira/software/c/projects/APIM/issues/?jql=project%20%3D%20%22APIM%22%20and%20fixVersion%20%3D%20${versions.branch}%20and%20status%20%3D%20Done%20ORDER%20BY%20created%20DESC)
+`;
+echo(chalk.blue('# Create PR on Github Doc repository'));
+echo(prBody);
```

Comment:
Maybe not necessary to print the description of the PR. We will see it in the PR.

Reply:
after discussion, you can leave it for debug purpose